src/text/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { v, create } from '@dojo/framework/core/vdom';
+import theme from '../middleware/theme';
+
+import * as css from '../theme/default/text.m.css';
+
+export interface TextProperties {
+	size?: 'xs' | 's' | 'm' | 'l' | 'xl' | 'xxl';
+	inverse?: boolean;
+	truncated?: boolean;
+	as?: string;
+}
```

Comment:
Have you considered adding `weight` and `uppercase` ? I've found those useful on recent projects.

Reply:
I think uppercase would be useful as a boolean that adds a text transform - Can you propose would you are thinking around weight?

Reply:
weight would follow the same as `size` ie. `s | m | l` etc and would apply `font-weight: 400 / 600 / 800` etc depending on the theme.